src/app/base/components/sidebar-pm/sidebar-pm.component.ts

```diff
@@ -161,4 +188,36 @@ export class SidebarPmComponent implements OnInit {
 
     }
   }
+
+  parseUTC(utc: string) {
+    const utcDate = dateAsUTCDate(new Date(utc));
+    return utcDate.toLocaleString();
+  }
+
+  howToUpdate() {
+    this.versionOverviewModal.nativeElement.checked = false;
+    this.stepsUpdate.nativeElement.checked = true;
+  }
+
+  back() {
+    this.stepsUpdate.nativeElement.checked = false;
+    this.versionOverviewModal.nativeElement.checked = true;
+  }
+
+  toggleTabs(index: number) {
+    this.openTab = index;
+  }
+
+  copyToClipboard(textToCopy) {
+    navigator.clipboard.writeText(textToCopy);
+  }
+
+  checkIfManagedVersion() {
+    if (!ConfigManager.isInit()) {
+      timer(250).subscribe(() => this.checkIfManagedVersion());
+      return;
+    }
+    this.isManaged = ConfigManager.getIsManaged();
+    this.versionOverview = ConfigManager.getVersionOverview();
```

Comment:
not sure why this is used with the config manager. Especially since the value in the config manager is never replaced with the newer requests in the subscription event.